### PR TITLE
fix: Crash when contact us pressed from account details

### LINKED ISSUE #1104: App crashes when Contact Us number clicked from the Savings Account and back button pressed.
**Summary:**

 App crashes when Contact Us number clicked from the Savings Account and back button pressed.

**Steps to reproduce:**

How can we reproduce the issue?
- Login to your mifos app
- Click on Accounts
- Click on any account
- Click on the Contact Us number at the bottom
- Press the back button

**Expected behavior:**

It should take user back to the account details page. This behavior is not seen in the main home page activity.  

**Observed behavior:**

The app crashes and when opened again it displays the home page

**Device and Android version:**

Redmi Y2

Android Version : 8.1.0

**Relevant GIF**

mifos

### PR DESCRIPTION
**Pull request by apoorva-mk**
Fixes #1104 

GIF of the app crash fixed.
ezgif-4-1ae49279129c

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by nikhil-sachdeva** [comment]
@apoorva-mk Please refrain from making changes in the gradle dependency files.

**Comment by apoorva-mk** [comment]
Yes I will revert the changes. I made the changes in my local repository as I was unable to build the app without upgrading the kotlin version used.

**Comment by apoorva-mk** [comment]
I have made all the changes and squashed the commits. Maintainers please review my PR  when you're free :)
@therajanmaurya @miPlodder

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#Tue Oct 10 11:28:53 IST 2017
+#Sat Mar 09 18:32:37 IST 2019
```
Comment (therajanmaurya) [inline gradle/wrapper/gradle-wrapper.properties]: Revert back this changes.

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (apoorva-mk) [reply gradle/wrapper/gradle-wrapper.properties]: @therajanmaurya I have made the requested changes. Please review the PR when you are free, thanks :)

**Comment by prashantkh19** [comment]
@apoorva-mk can you fix the merge conflicts?

**Comment by apoorva-mk** [comment]
@prashantkh19 I'm unable to build the app after migration to Android X, I'm looking into it

**Comment by prashantkh19** [comment]
#1178 fixes this. @therajanmaurya close this

**Pull request closed without merging**